### DISCUSSION TITLE
Thresher and Crawler eggs cannot be held

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#13929] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Tiger Thresher Eggs and Crawler Eggs have a defined base price and deconstruction materials, just like Mudraptor Eggs, but unlike Mudraptor Eggs they cannot be held or stored in a player's inventory. This means there is no way of deconstructing or selling them.

```
identifier="tigerthresheregg"

<Price baseprice="370" sold="false"/>
<Deconstruct time="10">
  <Item identifier="adrenalinegland" />
  <Item identifier="alienblood" />
</Deconstruct>
```
```
identifier="crawleregg"

<Price baseprice="200" sold="false"/>
<Deconstruct time="10">
  <Item identifier="alienblood" />
  <Item identifier="sulphuricacid" />
</Deconstruct>
```

### Reproduction steps

1. Attempt to pick up a Thresher or Crawler Egg

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by AzureNocte** [comment]
I think this is intended, otherwise they could be pick up in nest missions.
